### PR TITLE
Allow multiple file uploads using drag & drop for RoomView

### PR DESCRIPTION
**Pull request by oivoodoo**
Hi everyone!

In the riot-web created the issue about multiple file uploads.

https://github.com/vector-im/riot-web/issues/1953

I did the tests locally by dragging multiple images to `RoomView`. No issues in the web console as well.

Thank you for review!

Signed-off-by: Alexandr Korsak <alex.korsak@gmail.com>

### CODE REVIEW

File: src/components/structures/RoomView.js [hunk 1]
```diff
@@ -45,6 +45,8 @@ import RoomViewStore from '../../stores/RoomViewStore';
 import RoomScrollStateStore from '../../stores/RoomScrollStateStore';
 import SettingsStore from "../../settings/SettingsStore";
 
+import _every from "lodash/every";
+
```
Comment (t3chguy) [inline src/components/structures/RoomView.js]: any chance you could use the `Array.prototype.every` instead of lodash? They look identical to me

Comment (t3chguy) [reply src/components/structures/RoomView.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every

**Comment by t3chguy** [comment]
truth be told, I didn't even realise we had lodash as a dep

**Comment by t3chguy** [comment]
Apart from that nit, and sans testing it LGTM

**Comment by t3chguy** [comment]
Though https://jsperf.com/array-prototype-every-vs-every seems to point to lodash being faster for this scenario, argh

**Comment by t3chguy** [comment]
I shall test

**Comment by oivoodoo** [comment]
@t3chguy I have already replaced lodash.every by array.every. what's the best option do you think? I can revert it in case if lodash planned to be removed in the future.

**Comment by t3chguy** [comment]
works great :D

**Pull request merged**